emscripten.py

```diff
@@ -2413,6 +2413,11 @@ def create_asm_consts_wasm(forwarded_json, metadata):
                      str(int(sync_proxy)) +
                      ', code, sigPtr, argbuf); }')
 
+    if shared.Settings.RELOCATABLE:
+      # TODO(sbc): remove this conidtion after
+      # https://github.com/WebAssembly/binaryen/pull/2408 lands
+      preamble += '\n  if (core > %s) code -= %s;\n' % (shared.Settings.GLOBAL_BASE, shared.Settings.GLOBAL_BASE)
```

Comment:
Nit: in general can use `'{0} foo {0}'.format(bar)` to deduplicate str format args

Not really needed for this temp thing so lgtm as-is